.gitmodules

```diff
@@ -40,3 +40,6 @@
 [submodule "plugins/carlabase/carla"]
 	path = plugins/carlabase/carla
 	url = https://github.com/falktx/carla
+[submodule "src/3rdparty/jackaudio/jack"]
```

Comment:
@falkTX wrote:
> the right submodule is actually https://github.com/jackaudio/headers

Done via b852863, however now AppVeyor throws the following:

```
C:\projects\lmms\src\3rdparty\jackaudio\jack/types.h(25): fatal error C1083: Cannot open include file: 'pthread.h':
```

Which seems to be referenced here:
https://github.com/jackaudio/headers/blob/master/types.h

... but not here:
https://github.com/jackaudio/jack2/blob/develop/common/jack/types.h

Is this just a matter of the headers repo needing updating?  I'll also bump weakjack for MSVC support incase that's related.

Reply:
Per @falkTX, this needs to be reverted as the `headers` repo is based off of jack1, not jack2 and has issues compiling on MSVC.  Done via
f55f42d.